lib/measured/conversion.rb

```diff
@@ -48,6 +48,9 @@ def conversion_table
 
   def unit_for(name)
     unit = @units.find { |unit| unit.names_include?(name, case_sensitive: @case_sensitive) }
-    unit or raise Measured::UnitError, "Cannot find unit for #{name}"
+  end
+
+  def unit_for!(name)
+    unit_for(name) or raise Measured::UnitError, "Cannot find unit for #{name}"
```

Comment:
Fair enough. Again, I'm not in total agreement, but it's probably a good idea for us to stay in line with community style guides right now and change it upstream.

Just to be clear about why I'm in disagreement:

1. We're conflating boolean operators with control flow structures (unfortunately I think this is a symptom of many high-level programming languages).
2. The example from the style guide is actually the less intuitive structure (it should be `document.save! unless document.saved?`).
3. It seems weird, almost contradictory, to allow writing `do_stuff if foo = bar(baz)` but not `foo = bar(baz) or raise BarFailedError`.